src/main/java/com/Saurabh/GitHubAiAPI/StringOperations.java

```diff
@@ -6,7 +6,7 @@
 
 public class StringOperations {
 
```

Comment:
There are typos in the print statement. 'Invald' should be 'Invalid' and 'enter non-empty string' should be 'enter a non-empty string'.